### PR TITLE
Add redis

### PR DESCRIPTION
**Pull request by jkotalik**

### CODE REVIEW

File: src/Bicep.Core/TypeSystem/Radius/CommonBindings.cs [hunk 1]
```diff
@@ -57,6 +57,17 @@ public static class CommonBindings
             additionalPropertiesType: null,
             additionalPropertiesFlags: TypePropertyFlags.None);
 
+        public static NamedObjectType BindingRedis = new NamedObjectType(
```
Comment (rynowak) [inline src/Bicep.Core/TypeSystem/Radius/CommonBindings.cs:60]: Add this to the list on L122

File: src/Bicep.Core/TypeSystem/Radius/KnownTypes.cs [hunk 2]
```diff
@@ -209,6 +209,27 @@ public static NamedObjectType MakeServiceBusQueue()
             return MakeComponentType(kind, config: configType, bindings: bindings);
         }
 
+        public static NamedObjectType MakeRedis()
+        {
+            var kind = "redislabs.com/Redis@v1alpha1";
```
Comment (rynowak) [inline src/Bicep.Core/TypeSystem/Radius/KnownTypes.cs:215]: Add this to the list on Line 84

File: src/Bicep.Core/TypeSystem/Radius/KnownTypes.cs [hunk 3]
```diff
@@ -209,6 +209,27 @@ public static NamedObjectType MakeServiceBusQueue()
...
+            var configType = new NamedObjectType(
+                "config",
+                validationFlags: TypeSymbolValidationFlags.WarnOnTypeMismatch,
```
Comment (rynowak) [inline src/Bicep.Core/TypeSystem/Radius/KnownTypes.cs:218]: Also if you want to test, you can just F5 in vscode and it will run the extension for you.

**Comment by jkotalik** [comment]
Verified this is working.

**Review by rynowak — APPROVED** [review]

File: src/Bicep.Core/TypeSystem/Radius/CommonBindings.cs [hunk 4]
```diff
@@ -57,6 +57,17 @@ public static class CommonBindings
...
+            "binding: redislabs.com/Redis",
+            validationFlags: TypeSymbolValidationFlags.WarnOnTypeMismatch,
+            properties: new []
+            {
+                new TypeProperty("kind", new StringLiteralType("redislabs.com/Redis"), TypePropertyFlags.Required),
+                new TypeProperty("connectionString", LanguageConstants.String, TypePropertyFlags.None),
```
Comment (rynowak) [inline src/Bicep.Core/TypeSystem/Radius/CommonBindings.cs:66]: Did you end up adding any other values here? or just connection string?

Send another PR if you want to add more.
